airflow/providers/slack/hooks/slack.py

```diff
@@ -89,5 +90,9 @@ def call(self, api_method: str, **kwargs) -> None:
             form-encoding will take place. Optional.
         :param params: The URL parameters to append to the URL. Optional.
         :param json: JSON for the body to attach to the request. Optional.
+        :return: The server's response to an HTTP request. Data from the response can be
+            accessed like a dict.  If the response included 'next_cursor' it can be
+            iterated on to execute subsequent requests.
+        :rtype: SlackResponse
```

Comment:
```suggestion
```

Don’t need this since the return type is already visible in the function signature.